src/components/AgentDashboard(Rent)/ChangeStatus.js

```diff
@@ -29,16 +29,19 @@ function ChangeStatus() {
   console.log(propertyId);
 
   const [RenderRent, setRenderRent] = useState("rent");
-  const [RenderRentName, setRenderRentName] = useState("Rented on B8R");
+  const [RenderRentName, setRenderRentName] = useState("Rented of B8R");
```

Comment:
these should be "Rented on B8R" - @PrakharB8R, any reason you are replacing it? was this change approved?